config/routes.rb

```diff
@@ -3,13 +3,18 @@
 
   namespace :api do
     namespace :v1 do
+      delete 'users/:user_id/gyms/:gym_id', to: 'users/gyms/gym_members#destroy'
+      
```

Comment:
@tvaroglu @bfl3tch 

Thinking about this one further, if we want to be more RESTful, we could put this route in an Api::V1::GymMembersController with a `destroy` action and take the `user_id` and `gym_id` as query params.  So the main difference would be getting the ids from query params instead of from the path.  Aside from being more RESTful, it comes down to how we want to call it on the FE.

Same thought process could apply to `users/gyms/events#create` and `users/gyms/events#create`.

Let me know what you all think!

Reply:
One other thing to consider is security.  We don't want a malicious user to be able to delete records when they shouldn't be deleted.  But maybe that's another discussion :)